### PR TITLE
Add event listener for document.fonts.ready

### PR DESCRIPTION
**Pull request by stianpr**
When fonts take time to render we make sure to wait until they are
all loaded. We also bump beta version to 1.

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -128,10 +125,16 @@ app.use(async (ctx, next) => {
 
   logger.log('verbose', `Attempting to load ${url}`);
 
-  await page
-    .goto(url)
-    .then(screenshotDelay)
-    .catch(() => (gotoError = true));
+  try {
+    await page.goto(url);
+    await page.evaluateHandle('document.fonts.ready');
```
Comment (fdanielsen) [inline src/server.js:130]: Re: Our discussion about this. I actually made a little test verifying that it does actually await a property that is a promise too, it doesn't need to be a function that returns a promise. Made a test page the exposed a promise on a global variable, with a 10 second timeout before resolving, which I let the screenie server await in the same way as this does. So, should be good to go!

**Review by fdanielsen — APPROVED** [review]
Let's get this out, and test more heavily on our templates.

**Comment by asteinlein** [comment]
Did you test the potential issue mentioned at [the linked discussion](https://github.com/GoogleChrome/puppeteer/issues/422#issuecomment-380844283) about the event possibly not firing if the fonts are cached? Or do we need to add further protection for that case?

**Comment by fdanielsen** [comment]
> Did you test the potential issue mentioned at [the linked discussion](https://github.com/GoogleChrome/puppeteer/issues/422#issuecomment-380844283) about the event possibly not firing if the fonts are cached? Or do we need to add further protection for that case?

I didn't do any extensive caching testing, but what you referenced is not what we're using here. The comment you link to makes use of the dev tools API, while we're just awaiting the resolve of the `document.fonts.ready` promise which in my tests both resolved with and without a webfont to load.

**Pull request merged**